<?xml version="1.0" encoding="UTF-8"?>
<testrun name="PermsOnReplyPage" project="spw_001" tests="5" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="sanity_ff.PermsOnReplyPage" incomplete="true">
    <testcase name="SecureNoattachs" classname="sanity_ff.PermsOnReplyPage" time="26.243">
      <failure>
        <expected/>
        <actual>true</actual>org.junit.ComparisonFailure: expected:&lt;[]&gt; but was:&lt;[true]&gt;&#13;
	at org.junit.Assert.assertEquals(Assert.java:125)&#13;
	at org.junit.Assert.assertEquals(Assert.java:147)&#13;
	at sanity_ff.PermsOnReplyPage.SecureNoattachs(PermsOnReplyPage.java:262)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:601)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="NonSecureGreaterThan25mb" classname="sanity_ff.PermsOnReplyPage" time="5.838"/>
    <testcase name="SecureLesserThan25mb" classname="sanity_ff.PermsOnReplyPage" incomplete="true"/>
    <testcase name="NonSecureLesserThan25mb" classname="sanity_ff.PermsOnReplyPage" incomplete="true"/>
    <testcase name="SecureGreaterThan25mb" classname="sanity_ff.PermsOnReplyPage" incomplete="true"/>
  </testsuite>
</testrun>
